jenkinsapi/label.py

```diff
@@ -39,7 +39,13 @@ def get_tied_jobs(self):
         """
         Get a list of jobs.
         """
-        return self.poll(tree='tiedJobs[name,url,color]')['tiedJobs']
+        if self.get_tied_job_names():
+            job_list = []
+            for job in self.get_tied_job_names():
+                job_list.append(self.get_jenkins_obj().get_job(job['name']))
+            return job_list
```

Comment:
This will return a list of Job objects and not just job names. The function name is misleading.